client/js/components/GameView/Card.vue

```diff
@@ -34,7 +34,12 @@
       :src="require(`../../img/cards/Glasses_${suitName}.png`)"
       :alt="`Glasses - $${cardName}`"
     />
-    <img v-else-if="isBack" :src="require('../../img/cards/card_back.png')" alt="back" />
+    <img
+      v-else-if="isBack"
+      :src="require('../../img/cards/card_back.png')"
+      class="opponent-cardBack"
```

Comment:
Stick to kebab-case for css classes
```suggestion
      class="opponent-card-back"
```

Reply:
Change has been made.